src/game/Strategic/Creature_Spreading.cc

```diff
@@ -148,134 +144,59 @@ static CREATURE_DIRECTIVE* NewDirective(UINT8 ubSectorID, UINT8 ubSectorZ, UINT8
 	return curr;
 }
 
-
-static void InitLairDrassen(void)
+static void InitCreatureLair(const CreatureLairModel* lairModel)
 {
-	CREATURE_DIRECTIVE *curr;
-	giLairID = 1;
-	//initialize the linked list of lairs
-	lair = NewDirective( SEC_F13, 3, QUEEN_LAIR );
```

Comment:
If I see correctly, nothing uses `NewDirective` any more, so it can go.

Reply:
It is still used in ``InitCreatureLair`